ImperatorToCK3/Source/CK3Outputter/outTitles.cpp

```diff
@@ -10,31 +10,36 @@
 
 void CK3::outputTitleHistory(const std::shared_ptr<Title>& title, std::ofstream& outputStream) {
 	outputStream << title->getName() << " = {\n";
-	if (title->holder == "0")
-		outputStream << "\t1.1.1 = { holder = 0 }\n";
-	else {
-		outputStream << "\t867.1.1 = {\n";
-		
-		const auto& deFactoLiege = title->getDeFactoLiege();
-		if (deFactoLiege)
-			outputStream << "\t\tliege = " << deFactoLiege->getName() << "\n";
-		
-		outputStream << "\t\tholder = " << title->holder << "\n";
+
+	outputStream << "\t867.1.1 = {\n";
 		
-		if (title->government)
-			outputStream << "\t\tgovernment = " << *title->government << "\n";
-
-		const auto& succLaws = title->getSuccessionLaws();
-		if (!succLaws.empty()) {
-			outputStream << "\t\tsuccession_laws = {\n";
-			for (const auto& law : succLaws) {
-				outputStream << "\t\t\t" << law << "\n";
-			}
-			outputStream << "\t\t}\n";
+	const auto& deFactoLiege = title->getDeFactoLiege();
+	if (deFactoLiege)
+		outputStream << "\t\tliege = " << deFactoLiege->getName() << "\n";
+	
+	outputStream << "\t\tholder = " << title->getHolder() << "\n";
+	
+	const auto& govOpt = title->getGovernment();
+	if (govOpt)
+		outputStream << "\t\tgovernment = " << *govOpt << "\n";
+
+	const auto& succLaws = title->getSuccessionLaws();
+	if (!succLaws.empty()) {
+		outputStream << "\t\tsuccession_laws = {\n";
+		for (const auto& law : succLaws) {
+			outputStream << "\t\t\t" << law << "\n";
 		}
+		outputStream << "\t\t}\n";
+	}
 
-		outputStream << "\t}\n";
+	if (title->getRank() != TitleRank::barony) {
```

Comment:
Wait, I thought only counties had development?

Reply:
```
k_jazira = {
	867.1.1 = {	change_development_level = 5 }
	1066.1.1 = { change_development_level = 8 }

}
```
If a county doesn't have a development level set in history, it inherits the level set for its de jure lieges. Although I should probably make the level explicit for all counties before overriding development from Imperator.